src/Nethermind/Nethermind.Synchronization.Test/UnpooledReadOnlyList.cs

```diff
@@ -0,0 +1,37 @@
+// SPDX-FileCopyrightText: 2024 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using Nethermind.Core.Collections;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Nethermind.Synchronization.Test;
+/// <summary>
+/// This is list that does not require disposing and is meant for test purposes.
+/// </summary>
+/// <typeparam name="T"></typeparam>
+internal class UnpooledReadOnlyList<T> : List<T>, IOwnedReadOnlyList<T>
```

Comment:
Is it needed?

Reply:
It was to avoid `using` everytime you new up a request or response in the tests. But if the convention is to go with using, i can remove it.

Reply:
It is good to be reminded that you need to dispose! :)